impl/src/test/java/com/okta/sdk/impl/resource/StubResource.java

```diff
@@ -31,8 +31,10 @@
     public final  MapProperty mapProp = new MapProperty("mapPropKey");
     public final  ResourceListProperty<NestedStubResource> resourceListProp = new ResourceListProperty<>("resourceListPropKey", NestedStubResource.class);
     public final  StringProperty stringProp = new StringProperty("stringPropKey");
+    public final  CharacterArrayProperty charArrayProp1 = new CharacterArrayProperty("charArrayPropKey1");
+    public final  CharacterArrayProperty charArrayProp2 = new CharacterArrayProperty("charArrayPropKey2");
 
-    private final Map<String, Property> propertyDescriptors = AbstractResource.createPropertyDescriptorMap(booleanProp, dateProp, enumListProp, enumProp, integerProp, doubleProp, listProp, mapProp, resourceListProp, stringProp);
+    private final Map<String, Property> propertyDescriptors = AbstractResource.createPropertyDescriptorMap(booleanProp, dateProp, enumListProp, enumProp, integerProp, doubleProp, listProp, mapProp, resourceListProp, stringProp, charArrayProp1, charArrayProp2);
```

Comment:
nit: Are there common lint rules you could impose to break this into a multiline assignment?

Reply:
![Lots of standards](https://imgs.xkcd.com/comics/standards.png)

Reply:
I like Python's PEP8, and PHP has something similar, Java has a few common code styles.